controllers/consoleplugin/config/static-frontend-config.yaml

```diff
@@ -473,6 +475,41 @@ columns:
     calculated: substract(column.CollectionTime,TimeFlowEndMs)
     default: false
     width: 5
+  - id: PktDropBytes
```

Comment:
isn't it going to have side effect on the console plugin, to have this new field (since it's also nested under Bytes)

Reply:
There is an impact on both table and side panel indeed:
- table allow the user to pick these fields (disabled by default)
- side panel display these fields separately wich need a small refactor to skip these as it's already included in the other ones

![image](https://github.com/user-attachments/assets/c8a37d01-ce41-4cc6-af04-74a6ab20aa78)

Reply:
see https://github.com/netobserv/network-observability-console-plugin/pull/621/commits/c57e3ec5f612c7a86eba5cd8dab152fd09d2f3df